api/src/main/java/io/opencensus/trace/Tracer.java

```diff
@@ -151,6 +152,78 @@ public final Scope withSpan(Span span) {
   }
 
   /**
+   * Wraps a {@link Runnable} so that it executes with the {@code span} as the current {@code Span}
+   * and saves the rest of the Context.
```

Comment:
I wonder if it would be better for this method to only set the current span.  Then it would compose better with a corresponding method for tags. The same applies to startSpanAndWrap.

Reply:
@sebright let me think more about this.
@adriancole any suggestion?